packages/design-system-web/src/components/geov-entity-class-label/geov-entity-class-label.tsx

```diff
@@ -101,6 +107,7 @@ export class GeovEntityClassLabel {
   render() {
     return (
       <Host>
+        {this.withIcon && <geov-entity-class-icon type={this.data.label?.toLowerCase().replace(/\s+/g, '')}></geov-entity-class-icon>}
```

Comment:
Instead of passing in the class label, pass in the class uri. To get the class URI, you have to properly store the URI from the sparql result in fetchData():
```javascript
// extend interface
export interface GeovClassLabelData extends FetchResponse {
  label?: string;
  classURI: string; // <- add this
  error?: boolean;
}
// instead of
 return sparqlJson<{ classLabel: SparqlBinding }>(this.sparqlEndpoint, qrLabel(this.entityId))
      .then(res => {
        return {
          ...this.data,
          label: res?.results?.bindings?.[0]?.classLabel?.value,
          loading: false,
        };
 

// write
 return sparqlJson<{ classLabel: SparqlBinding, t: SparqlBinding }>(this.sparqlEndpoint, qrLabel(this.entityId))
      .then(res => {
        return {
          ...this.data,
          label: res?.results?.bindings?.[0]?.classLabel?.value,
          classURI: res?.results?.bindings?.[0]?.t?.value, // <- value or uri, please check this.
          loading: false,
        };
```

then you can do

```javascript
// instead of 
{this.withIcon && <geov-entity-class-icon type={this.data.label?.toLowerCase().replace(/\s+/g, '')}></geov-entity-class-icon>}

// write
{this.withIcon && <geov-entity-class-icon classURI={this.data.classURI)}></geov-entity-class-icon>}
```